cumulus/components/userdata/linux.py

```diff
@@ -0,0 +1,38 @@
+from troposphere import (
+    Ref,
+    Join,
+)
+
+
+def user_data_for_cfn_init(launch_config_name, asg_name, configsets):
+    """
```

Comment:
Nothing urgent for now but we can get this into a flat file, and read it in.    You can still do substitutions that way,  you'd just run a `Sub(file.read()) ` , with vars in the `${var}` syntax..    but not urgent, just easier to maintain.

Reply:
pulled out into task #13